pkgs/development/python-modules/pypdt/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "PyPDT";
+  version = "0.8.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "92dc26389e61f7527e9ca296e58e4d58f979b2fd0a9ce40e9eeccf356c60ac13";
+  };
+
+  doCheck = false; # no tests
+  pythonImportsCheck = [ "pypdt" ];
+
+  meta = with lib; {
+    description = "Python package for high-energy and nuclear physics Particle Data Table and Particle ID code interpretation";
+    homepage = "https://gitlab.com/hepcedar/pypdt";
+    license = licenses.unfree;
```

Comment:
The `setup.py` file says [`GPL`](https://gitlab.com/hepcedar/pypdt/-/blob/master/setup.py#L20). It would be best to clarify it with upstream.

Reply:
Upstream issue: https://gitlab.com/hepcedar/pypdt/-/issues/1